src/components/DevPanel.tsx

```diff
@@ -0,0 +1,369 @@
+  { value: '2', label: '2' },
+  { value: '5', label: '5' },
+  { value: '10', label: '10' },
+  { value: '25', label: '25' },
+  { value: '50', label: '50' },
+];
+
+interface State {
+  pdfFiles: FileList | undefined,
+  buttonState: string
+}
+
+class DevPanel extends Component<Props, State, {}> {
+  handleChangeFileUpload(event: any, rowIndex: number) {
+    event.preventDefault();
+    const {
+      alert,
+    } = this.props;
+    const { files } = event.target;
+
+    this.setState({
+      pdfFiles: files,
+    }, () => this.handleFileChange(rowIndex));
+  }
+
+  handleFileChange(rowIndex: number) {
+    const {
+      alert,
+    } = this.props;
+
+    if (this.state.pdfFiles === undefined) throw new Error('Must upload a file');
+    const pdfFile = this.state.pdfFiles[0];
+    if (pdfFile === null) throw new Error('Must upload a file');
+
+    const formData = new FormData();
+    formData.append('file', pdfFile, pdfFile.name);
+    formData.append('pdfType', PDFType.FORM);
+    formData.append('fileId', this.state.documents[rowIndex].id);
+
+    fetch(`${getServerURL()}/upload`, {
+      method: 'POST',
+      credentials: 'include',
+      body: formData,
+    }).then((response) => response.json())
+      .then((responseJSON) => {
+        const {
+          status,
+        } = JSON.parse(responseJSON);
+        if (status === 'SUCCESS') {
+          alert.show(`Successfully uploaded ${pdfFile.name}`);
+          this.setState({
+            buttonState: '',
+            pdfFiles: undefined,
+          }, () => this.getDocuments());
+        } else {
+          alert.show(`Failure to upload ${pdfFile.name}`);
+          this.setState({ buttonState: '' });
+        }
+      });
+  }
+
+  handleChangeFileDownload(event: any, rowIndex: number) {
+    event.preventDefault();
+    const {
+      alert,
+    } = this.props;
+    const { files } = event.target;
+
+    this.setState({
+      pdfFiles: files,
+    }, () => this.handleFileDownload(rowIndex));
+  }
+
+  handleFileDownload(rowIndex: number) {
+    const {
+      userRole,
+    } = this.props;
+
+    const documentId = this.state.documents[rowIndex].id;
+    const documentName = this.state.documents[rowIndex].filename;
+
+    let pdfType;
+    if (userRole === Role.Worker || userRole === Role.Admin || userRole === Role.Director) {
+      pdfType = PDFType.FORM;
+    } else if (userRole === Role.Client) {
+      pdfType = PDFType.IDENTIFICATION;
+    } else {
+      pdfType = undefined;
+    }
+
+    fetch(`${getServerURL()}/download`, {
+      method: 'POST',
+      credentials: 'include',
+      body: JSON.stringify({
+        fileId: documentId,
+        pdfType,
+      }),
+    }).then((response) => response.blob())
+      .then((response) => {
+        const pdfFile = new File([response], documentName, { type: 'application/pdf' });
+        console.log(pdfFile);
+
+        const url = window.URL.createObjectURL(response);
+        const a = document.createElement('a');
```

Comment:
can we have better variable names than "a"

Reply:
This is the name when you make a href